tensorflow/compiler/xla/client/xla_builder.cc

```diff
@@ -289,6 +289,14 @@ Status XlaBuilder::SetDynamicBinding(int64 dynamic_size_param_num,
   return Status::OK();
 }
 
+Status XlaBuilder::AddFrontendAttribute(const XlaOp& op, std::string attribute,
```

Comment:
The declaration needs a comment that this overwrites existing attributes.

Reply:
Done